src/datasets/active_profiles.md

```diff
@@ -0,0 +1,140 @@
+# Intro
+
+The `active_profiles` dataset gives client-level estimates of whether a profile
+is still an active user of the browser at a given point in time, as well as probabilistic forecasts
```

Comment:
specify "desktop" browser, and any particular selection criteria you had? (all OS and locations, for example?)